src/core/engines/cross-margin/CrossMarginEngine.sol

```diff
@@ -196,6 +211,24 @@ contract CrossMarginEngine is
         delete accounts[_subAccount];
     }
 
+    /**
+     * @notice  sets the Partial Margin Mask for a pair of assets
+     * @param _assetX the id of the asset a
+     * @param _assetY the id of the asset b
+     * @param _value is margin-able
+     */
+    function setPartialMarginMask(address _assetX, address _assetY, bool _value) external {
+        _checkOwner();
+
+        uint256 collateralId = grappa.assetIds(_assetX);
+        uint256 mask = 1 << (grappa.assetIds(_assetY) & 0xff);
```

Comment:
what is the purpose of `&0xff`? isn't it just gonna be the same value?

Reply:
Taken from the oz bitmap implementation